lib/reporter.js

```diff
@@ -441,7 +456,7 @@ class Reporter {
     finishSuite () {
         const currentSuiteTempId = this.getCurrentSuiteTempId();
         const status = this.collectionRunOptions.collection &&
-            utils.getCollectionVariablesByKey('status', this.collectionRunOptions.collection.variables);
+            utils.getCollectionVariablesByKey('status', this.collectionRunOptions.collection.variables) || 'PASSED';
```

Comment:
The ReportPortal API will calculate the final suite status based on the statuses of its descendants, so setting here `PASSED` as a fallback may lead to wrong suite status in the system.